profiler/src/main/java/com/navercorp/pinpoint/profiler/instrument/PlainClassLoaderHandler.java

```diff
@@ -102,7 +102,9 @@ private ClassPool createClassPool(ClassLoader classLoader) throws NotFoundExcept
     private Class<?> injectClass0(ClassPool pool, ClassLoader classLoader, String className, ClassLoadingChecker classLoadingChecker) throws NotFoundException, IOException, CannotCompileException, IllegalArgumentException, IllegalAccessException, InvocationTargetException {
         Class<?> c = null;
         try {
-            c = classLoader.loadClass(className);
+            // Using different approach to load classes via classloader
+            // c = classLoader.loadClass(className);
+            c = Class.forName(className, false, classLoader);
```

Comment:
Hi @suraj-raturi 
I fixed the JBoss CL issue from #2117
But This PR need to modify setting.
**plugins/jboss/README.md**
 ~~-Djboss.modules.system.pkgs=com.navercorp.pinpoint~~
 -Djboss.modules.system.pkgs=com.navercorp.pinpoint.bootstrap,
                                                   com.navercorp.pinpoint.common,
                                                   com.navercorp.pinpoint.exception